src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -610,7 +654,7 @@ private void spawnEntity(@Nullable Entity entity)
     {
         if (entity != null)
         {
-            BlockPos pos = job.getSchematic().getOffsetPosition();
+            BlockPos pos = job.getStructure().getOffsetPosition();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'pos' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)